dotCMS/src/main/java/com/dotmarketing/cms/myaccount/action/AddressAction.java

```diff
@@ -1,9 +1,11 @@
 package com.dotmarketing.cms.myaccount.action;
 
 import com.dotcms.repackage.org.apache.struts.actions.DispatchAction;
+import com.dotmarketing.cms.SecureAction;
 
 
-public class AddressAction extends DispatchAction {
+@Deprecated
+public class AddressAction extends SecureAction {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)